### PR TITLE
fix(folders): fix root level folders generation issue

### PR DESCRIPTION
**Pull request by yuvarajuv**
### Problem

Root level folders are not created that are merged to the first root folder.

### Fix:
 
Fixed root level folders generation issue

### CODE REVIEW

File: utils/path.js [hunk 1]
```diff
@@ -13,11 +13,16 @@ function initStore(initialDirectory = ['blocks']) {
     basePath.pop()
   }
 
+  function resetAllPaths(){
+    basePath.splice(0, basePath.length)
```
Comment (ramsunvtech) [inline utils/path.js:17]: can this be with slice method.

```
let basePath = [...initialDirectory]
......
basePath = basePath.slice(0, basePath.length)
```

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: utils/path.js (hunk 1 above)
Comment (yuvarajuv) [reply utils/path.js:17]: No. It is not working as expected if we use the above code.

Comment (ramsunvtech) [reply utils/path.js:17]: my point is on mutation. nm will have to revisit later on the issue #16 16

Comment (ramsunvtech) [reply utils/path.js:17]: LGTM for now

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**